### PR TITLE
nova-filters: migrate to scons_latest

### PR DESCRIPTION
**Pull request by wegank**
###### Description of changes

#201859

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/nova-filters/default.nix [hunk 1]
```diff
@@ -9,20 +9,22 @@ stdenv.mkDerivation {
...
 
   patchPhase = ''
-    # remove TERM:
-    sed -i -e '4d' SConstruct
-    sed -i "s@mfpmath=sse@mfpmath=sse -I ${boost.dev}/include@g" SConstruct
-    sed -i "s@ladspa.h@${ladspaH}/include/ladspa.h@g" filters.cpp
-    sed -i "s@LADSPA_HINT_SAMPLE_RATE, 0, 0.5@LADSPA_HINT_SAMPLE_RATE, 0.0001, 0.5@g" filters.cpp
-    sed -i "s/= check/= detail::filter_base<internal_type, checked>::check/" nova/source/dsp/filter.hpp
+    # remove TERM and Options:
+    sed -i -e '4d;7,17d' SConstruct
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/nova-filters/default.nix:17]: Can we do a patch instead? This could easily break with the next update.

Comment (wegank) [reply pkgs/applications/audio/nova-filters/default.nix:17]: The package was from 2008 and never updated, so this should be fine...

Comment (wegank) [reply pkgs/applications/audio/nova-filters/default.nix]: I've made the diff lighter.

**Comment by thiagokokada** [comment]
Result of `nixpkgs-review pr 209248` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>nova-filters</li>
  </ul>
</details>

**Review by thiagokokada — APPROVED** [review]

**Comment by magnetophon** [comment]
@wegank Thank you!  I tried, but was not able to do this.

**Pull request merged**